AspNetCoreWeb/Infrastructure/SearchOptionsProcessor.cs

```diff
@@ -77,7 +77,7 @@ private static IEnumerable<SearchTerm> GetValidTerms(IEnumerable<DTColumn> colum
         public static IQueryable<TEntity> Apply(IQueryable<TEntity> query, IEnumerable<DTColumn> columns)
         {
             var terms = GetValidTerms(columns).ToArray();
-            if (terms.Any())
+            if (!terms.Any())
```

Comment:
@JudeVajira , I checked my `master` branch and line 80 is already having a negate operator. `if (!terms.Any())` How did your commit has `if (terms.Any())` ?